client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/coverage/methodreplacement/thirdpartyclasses/NettyHttpUtilClassReplacement.java

```diff
@@ -0,0 +1,41 @@
+package org.evomaster.client.java.instrumentation.coverage.methodreplacement.thirdpartyclasses;
+
+import org.evomaster.client.java.instrumentation.coverage.methodreplacement.Replacement;
+import org.evomaster.client.java.instrumentation.coverage.methodreplacement.ThirdPartyMethodReplacementClass;
+import org.evomaster.client.java.instrumentation.coverage.methodreplacement.UsageFilter;
+import org.evomaster.client.java.instrumentation.shared.ReplacementType;
+import org.evomaster.client.java.instrumentation.staticstate.ExecutionTracer;
+import org.evomaster.client.java.utils.SimpleLogger;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class NettyHttpUtilClassReplacement extends ThirdPartyMethodReplacementClass {
+
+    private static final NettyHttpUtilClassReplacement singleton = new NettyHttpUtilClassReplacement();
+
+    @Override
+    protected String getNameOfThirdPartyTargetClass() {
+        return "io.netty.handler.codec.http.HttpUtil";
+    }
+
+    @Replacement(replacingStatic = true,
+            type = ReplacementType.TRACKER,
+            id = "setKeepAlive_void_class",
+            usageFilter = UsageFilter.ONLY_SUT)
+    public static Object setKeepAlive(Object caller, Object h, Object httpVersion, boolean keepAlive) {
+        if(caller == null){
+            throw new NullPointerException();
+        }
+
+        Method original = getOriginal(singleton, "setKeepAlive_void_class", caller);
+
+        try {
+            return original.invoke(caller, h, httpVersion, true);
```

Comment:
add a comment like: "we ignore the keepAlive input, and always call the original code with true"